binding-library/python/setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
+
+setup(
+    name='azure-functions-kafka-trigger',
```

Comment:
Would be more appropriate to name this `azure-functions-kafka-binding` since you also have `In` and `Out` converters defined. I don't think the scenarios for in and out bindings are in play right now but would be good to have a generic package name that can encompass all these scenarios.